File: src/main/java/ejerciciosJava/Ejercicio11.java
package ejerciciosJava;

public class Ejercicio11 {
    public static void main() {
        int num = Utils.pedirEntero();
        int a = 0, b = 1;

        System.out.println("Los primeros " + num + " números de la secuencia de Fibonacci son:");
        for (int i = 1; i <= num; i++) {
            System.out.print(a + " ");
            int temp = a;
            a = b;
            b += temp;
        }
    }
}

File: src/main/java/ejerciciosKotlin/Ejercicio11.kt
package ejerciciosKotlin

fun ejercicio11(){

    val num = pedirEntero()

    var a = 0
    var b = 1

    println("Los primeros $num números de la secuencia de Fibonacci son:")
    for (i in 1..num) {
        print("$a ")
        val temp = a
        a = b
        b += temp
    }
}